config/webpack.cozyui.config.js

```diff
@@ -13,9 +13,9 @@ module.exports = {
       {
         test: /\.styl$/,
         loader: ExtractTextPlugin.extract([
-          'css-loader?importLoaders=1',
+          'css-loader?importLoaders=1&modules',
           'postcss-loader',
-          'stylus-loader'
+          'stylus-loader?paths=node_modules/cozy-ui/stylus/'
```

Comment:
@CPatchane This line is from @m4dz so i'm not really sure what it does, but i think it just gives a styl folder to look up to for Stylus, on top of the folder Stylus is already watching (.styl files from the app in this case).